cmd/main.go

```diff
@@ -139,6 +157,10 @@ func main() {
 		FeatureOptions:   featureOptions,
 	}
 
+	if *runController && *enableMultishare && *enableStateful {
+		runMultishareReconciler(config)
```

Comment:
why are we starting the reconciler in every replica? It should be started when the driver replica wins the leader election?

Reply:
added TODO within `runMultishareReconciler` to fix in the next PR